src/cpp/session/http/SessionHttpConnectionListenerImpl.hpp

```diff
@@ -96,6 +75,10 @@ class HttpConnectionListenerImpl : public HttpConnectionListener,
 
       // accept next connection (asynchronously)
       acceptNextConnection();
+      
+      // initialize and refresh locks
+      core::FileLock::initialize();
```

Comment:
Don't we already call initialize here: https://github.com/rstudio/rstudio/blob/feature/directory-file-locking/src/cpp/session/SessionMain.cpp#L1660 ?